lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -316,7 +316,15 @@ end
 
 e2function void entity:propGravity(number gravity)
 	if not PropCore.ValidAction(self, this, "gravity") then return end
-	PropCore.PhysManipulate(this, nil, nil, nil, gravity, nil)
+	local physCount = this:GetPhysicsObjectCount()
```

Comment:
Need isowner check I think

Reply:
I guess change the 'ValidAction' to 	`if not isOwner(self, this) then return end`